weave/execute.py

```diff
@@ -450,10 +450,13 @@ def get_bytes_read_to_arrow(node: graph.Node, result: typing.Any) -> int:
     if (
         isinstance(node, graph.OutputNode)
         and node.from_op.name in op_policy.ARROW_FS_OPS
-        and isinstance(result, ArrowWeaveList)
     ):
-        return result._arrow_data.nbytes
-
+        if node.from_op.name.startswith("mapped"):
+            return sum(
```

Comment:
Can this be a generator expression (i.e. remove the square brackets)

Reply:
done